src/azure-cli/azure/cli/command_modules/ams/_client_factory.py

```diff
@@ -85,3 +85,7 @@ def get_account_filters_client(cli_ctx, *_):
 
 def get_asset_filters_client(cli_ctx, *_):
     return cf_media(cli_ctx).asset_filters
+
+
+def sync_storage_keys(cli_ctx, *_):
```

Comment:
Could you tell the purpose that you defined it as `sync_storage_keys` instead of `media_services` which is more meaningful?